pkgs/development/python-modules/altair/default.nix

```diff
@@ -22,7 +22,7 @@
 
 buildPythonPackage rec {
   pname = "altair";
-  version = "5.4.1";
+  version = "5.5.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
```

Comment:
3.8 support was dropped in this version. See https://github.com/vega/altair/releases/tag/v5.5.0

So this should probably be `pythonOlder "3.9"`? That said, I am not sure there's a reason to have this attribute set at all, since we only have `python39Packages` and up. Maybe remove it?

Reply:
By my understanding we keep these seemingly useless pythonOlder statements to benefit users who mix Python 3.8 (or older) from some pinned nixpkgs version, whilst still using (some) new packages.

Reply:
@FliegendeWurst interesting. Maybe this should be clarified in python docs since it's not obvious.